dashboard/test/factories/factories.rb

```diff
@@ -537,7 +537,10 @@
     end
 
     trait :script do
-      create(:script_level)
+      after :create do |level|
+        script_level = create(:script_level, levels: [level])
+        create(:lesson_group, lessons: [script_level.lesson], script: script_level.script)
+      end
```

Comment:
what do you think about calling this ` trait :with_script`, to match some of the other traits which add associated objects? I think it would improve readability, but it can be up to you whether this is worth the trouble.

Reply:
I'm down! I kept it as just `script` to preserve existing behavior, but I agree that `with_script` reads much better. I think I'll pursue that in a follow-up PR, though, since I expect I'll have to update existing tests to do.